app/models/project.rb

```diff
@@ -3,10 +3,16 @@ class Project < ActiveRecord::Base
 
   validates :name, :repository, :pivotal_id, presence: true
 
+  has_many :code_climate_scores
+
   def self.ordered_by_name
     order(:name)
   end
 
+  def self.with_repository(repository)
+    where(repository: repository)
```

Comment:
could remove the `first` call from the importer if you make this `find_by_repository(repository)` if we can safely assume each project has a different repo - maybe that's too limiting though